pkgs/applications/editors/kakoune/plugins/overrides.nix

```diff
@@ -131,6 +132,41 @@ declare-option -hidden str ansi_filter %{'"$out"'/bin/kak-ansi-filter}
     '';
   });
 
+  hop-kak = rustPlatform.buildRustPackage rec {
+    pname = "hop-kak";
+    version = "0.2.0";
+
+    src = fetchgit {
+      url = "https://git.sr.ht/~hadronized/hop.kak";
+      rev = "7314ec64809a69e0044ba7ec57a18b43e3b5f005";
+      sha256 = "/J1iOkM+fkWXGYmzSS72mcBjY2Ew9Rx48I6cO+wlk3E=";
+    };
+    cargoHash = "sha256-EjSj/+BysGwJBxK6Ccg2+pXHdB2Lg3dxIURRsSVTHVY=";
+
+    # hop.kak uses the current git rev as the version info under
+    # the help flag. First, patch out the git call...
+    patches = [ ./patches/hop-kak-no-git.patch ];
```

Comment:
That will enable you to drop this patch, I believe.

Reply:
and the sed as well. thanks!